src/Core/src/Handlers/View/AbstractViewHandler.MaciOS.cs

```diff
@@ -13,7 +14,12 @@ public partial class AbstractViewHandler<TVirtualView, TNativeView> : INativeVie
 		public void SetFrame(Rectangle rect)
 		{
 			if (View != null)
+			{
 				View.Frame = rect.ToCGRect();
+
+				if (VirtualView?.Background?.IsNullOrEmpty() == false)
+					View.UpdateBackgroundLayerFrame();
```

Comment:
iOS is special in that the layers need to be resize when the control is resized.